google/cloud/spanner_v1/_helpers.py

```diff
@@ -228,6 +228,11 @@ def _parse_value_pb(value_pb, field_type):
             return float(value_pb.string_value)
         else:
             return value_pb.number_value
+    elif type_code == TypeCode.FLOAT32:
```

Comment:
We have integration tests for every type in [test_session_api.py](https://github.com/googleapis/python-spanner/blob/e73c6718b23bf78a8f264419b2ba378f95fa2554/tests/system/test_session_api.py#L2182). Can we have a test for `FLOAT32`?
Reference: `test_execute_sql_w_float64_bindings` and `test_execute_sql_w_float_bindings_transfinite`

The comment mentioned [below](https://github.com/googleapis/python-spanner/pull/1113/files#r1518576993) can be verified through the test `test_execute_sql_w_float_bindings_transfinite`